vllm/model_executor/models/llama.py

```diff
@@ -364,6 +433,18 @@ def load_weights(self,
                 loaded_weight = convert_pyslice_to_tensor(loaded_weight)
                 loaded_weight = loaded_weight.T
 
+            if int8_fusion:
+                is_fusion_weight = False
+                for weight_name, _ in self._int8_scale_params.items():
+                    if weight_name not in name:
+                        continue
+                    param = state_dict[name.replace(
+                        weight_name, self._int8_scale_params[weight_name])]
+                    param.copy_(loaded_weight)
```

Comment:
this failed with `TypeError: copy_(): argument 'other' (position 1) must be Tensor, not builtins.PySafeSlice`. this is because `loaded_weight` which is type PySafeSlice not converted to tensor. for 0-dim scalers, i can't find a way to do this conversion (can't even get the value out). however it works by updating `f.get_slice` to use `f.get_tensor` in `hf_model_weights_iterator`. I'm not sure what's the difference between `get_slice` and `get_tensor`?

alternatively i think this could be fixed if these values are serialized as a dim-1 tensor

Reply:
I am looking into the problem. Does this happen in each iteration or in specific params?

Reply:
params like `.a`, `.post_attention_layernorm.weight` and `.inscale` which are scalars